app/helpers/identifications_helper.rb

```diff
@@ -1,18 +1,13 @@
 module IdentificationsHelper
   DFE_SIGN_IN_OPTIONS = [
-    OpenStruct.new(name: 'milton_keynes', to_radio: ['milton_keynes', 'Milton Keynes'])
-  ].freeze
-
-  AZURE_SIGN_IN_OPTIONS = [
+    OpenStruct.new(name: 'milton_keynes', to_radio: ['milton_keynes', 'Milton Keynes']),
     OpenStruct.new(name: 'cambridgeshire', to_radio: ['cambridgeshire', 'Cambridgeshire']),
     OpenStruct.new(name: 'the_north_east', to_radio: ['the_north_east', 'The North East'])
   ].freeze
 
   def identification_options
     @identification_options ||= begin
-      opts = AZURE_SIGN_IN_OPTIONS.map(&:to_radio)
-      opts += DFE_SIGN_IN_OPTIONS.map(&:to_radio) unless Rails.env.production?
```

Comment:
Without this guard we will be putting the Milton Keynes into production here and we have since changed plan. This was handy to help us test and prototype DSI but now that we have 2 real options for use with DSI it's no longer needed.

Reply:
Added a commit that removes Milton Keynes ✨